examples/solvers/chambolle_pock_denoising_complex.py

```diff
@@ -0,0 +1,83 @@
+"""Total variation denoising of complex data using the Chambolle-Pock solver.
+
+Solves the optimization problem
+
+    min_{x}  1/2 ||x - g||_2^2 + lam || |grad(x)| ||_1
+
+Where ``grad`` the spatial gradient and ``g`` is given noisy data.
+
+For further details and a description of the solution method used, see
+:ref:`chambolle_pock` in the ODL documentation.
+"""
+
+import numpy as np
+import scipy
+import odl
+
+# Read test image: use only every second pixel, convert integer to float,
+# and rotate to get the image upright
+dtype = np.complex64
+image = np.rot90(scipy.misc.ascent()[::1, ::1], 3).astype(dtype)
```

Comment:
This `astype` is unnecessary since the next line will implicitly cast to complex

Reply:
I can remove the `dtype` variable, but .astype('float32') will be needed if we want to end up as 'complex64' and not 'complex128'